stargz/reader/reader.go

```diff
@@ -39,7 +39,8 @@ import (
 )
 
 const (
-	PrefetchLandmark = ".prefetch.landmark"
+	PrefetchLandmark   = ".prefetch.landmark"
+	NoPrefetchLandmark = ".no.prefetch.landmark"
```

Comment:
Make sure to document these files

Reply:
I'll add a doc about e-SGZ structure with another PR.